articles/azure-maps/creator-qgis-plugin.md

```diff
@@ -0,0 +1,221 @@
+---
+title: View and edit data with the Azure Maps QGIS plugin
+titleSuffix: Microsoft Azure Maps Creator
+description: How to view and edit indoor map data using the Azure Maps QGIS plugin
+author: brendansco
+ms.author: Brendanc
+ms.date: 05/31/2023
+ms.topic: how-to
+ms.service: azure-maps
+services: azure-maps
+---
+
+# View and edit data with the Azure Maps QGIS plugin
+
+[QGIS] is an open-source [geographic information system (GIS)] application that supports viewing, editing, and analysis of geospatial data.
+
+[Azure Maps Creator] is used to develop applications based on your private indoor map data using Azure Maps API and SDK. You can use building blueprints in CAD format, or geojson format to create indoor maps using Azure Maps Creator. Map data is stored in the [Dataset Service], can be accessed and edited using the [Features] service and viewed using the [Tileset] and [Style] services.
```

Comment:
I think we can leave this paragraph out. We can just have a link to Azure Maps Creator and link to the concepts page somewhere from here.

Reply:
removed